### PR TITLE
Add sleep in :wait-interpolation-until loop

### PR DESCRIPTION
**Pull request by pazeshun**
https://github.com/start-jsk/jsk_apc/pull/2235#discussion_r126157616

### CODE REVIEW

File: jsk_arc2017_baxter/euslisp/lib/baxter-interface.l [hunk 1]
```diff
@@ -375,7 +376,8 @@
       (while (send self :interpolatingp)
         (when (reduce #'(lambda (x y) (or x y)) (mapcar #'funcall conds))
           (ros::ros-info "[:wait-interpolation-until] Cancel angle vector: ~a" arm)
-          (send self :cancel-angle-vector)))))
+          (send self :cancel-angle-vector))
+        (unix::usleep 1000))))
```
Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/baxter-interface.l]: BTW, Do we really need these lines? lime373-375
```
;; wait for :interpolatingp
    (dotimes (x 100)
      (if (send self :interpolatingp) (return))
      (unix::usleep 1000))
```

**Comment by knorth55** [comment]
I have no idea why you need to `unix::usleep`.
Write more detailed information for review.

**Comment by knorth55** [comment]
Is this for waiting `cancel-angle-vector` execution?

File: jsk_arc2017_baxter/euslisp/lib/baxter-interface.l (hunk 1 above)
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/baxter-interface.l]: Just after sending angle-vector, interpolatingp sometimes return nil even if robot moves correctly

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/baxter-interface.l]: ok

**Comment by pazeshun** [comment]
It's for reducing CPU load.
Without sleep, one CPU load becomes nearly 100%, even if we don't need such power to detect signal of 100 Hz

**Comment by knorth55** [comment]
Ok now i understand.
Please write more detailed info if you request a review unless it is quite troublesome to find out the context.

**Comment by pazeshun** [comment]
Sorry

**Pull request merged**